### PR TITLE
add gpu test notebook

### PR DESCRIPTION
**Pull request by MichaelClifford**
Short notebook to confirm JH is using GPU's

### CODE REVIEW

**Review by erikerlandson — APPROVED** [review]

**Comment by MichaelClifford** [comment]
@erikerlandson I added a bit more content to this notebook. Take a look and let me know if its ready to merge : )

File: notebooks/confirm_gpu_available.ipynb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+   "source": [
+    "# Confirm Notebook Can Access GPU \n",
+    "\n",
+    "This notebook provides a couple of small examples that confirm your notebook environment is able to connect to a GPU, either locally or on a cluster. The bottom section also provides a short example showcasing the increased performance of using the GPU over a CPU for a matrix multiplication task. \n",
+    "\n",
+    "\n",
+    "The one caveat to ensuring this notebook works correctly and you can communicate with the gpu, is that you must install pytorch.\n",
+    "\n",
+    "```\n",
+    "pip3 install torch==1.9.0+cu114 torchvision==0.10.0+cu114 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
```
Comment (chauhankaranraj) [inline notebooks/confirm_gpu_available.ipynb:16]:
```suggestion
    "pip3 install torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
```
Hmm I only see cuda 11.1 wheels on https://download.pytorch.org/whl/torch_stable.html. Running this pip install gives me an error:

Screenshot from 2021-09-22 10-57-59

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
Looks great! :clap: Just left a small comment

File: notebooks/confirm_gpu_available.ipynb (hunk 1 above)
Comment (MichaelClifford) [reply notebooks/confirm_gpu_available.ipynb:16]: great catch @chauhankaranraj that is absolutely the wrong request. :)

**Review by chauhankaranraj — APPROVED** [review]

**Comment by erikerlandson** [comment]
:tada:

**Pull request merged**